.editorconfig

```diff
@@ -16,3 +16,7 @@ indent_size = 4
 
 [**.{yml,yaml}]
 indent_size = 2
+
```

Comment:
what is this file for?

Reply:
Provide basic configuration to editors about how text should be formatted.
Several IDEs evaluate this file to configure workspace accordingly for unified text formatting, e. g. indention, encoding, EOL, etc.
See https://editorconfig.org/